File: irenaposkiene-cs50-2018-spring-mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int a;
    // ask for a height
    do
    {
        a = get_int("HEIGHT:");

    }
    while (a < 0 || a > 23);

        // Print lines
    for (int i = 0; i < a; i++)
    {



        for (int j = 0; j  < a - 1 - i ; j++)
        {

            printf(" ");

        }
        // print hashes
        for (int z = 0; z < i + 1 ; z++)
        {

            printf("#");


        }

        // print spaces

        for (int j = 0; j < 2; j++)
        {

            printf(" ");

        }
        // print hashes
        for (int z = 0; z < i + 1; z++)
        {
            printf("#");

        }

        printf("\n");
    }


}

File: irenaposkiene-cs50-2018-spring-sentimental-hello/mario.py
import cs50

def main():


    # ask for a height

    while True:
        print("Height: ", end = "")

        a = cs50.get_int()
        if a <= 23 and a > 0:
            break

    # Print lines
        for i in range(0, a):

            for j in range(0, a-1-i):

                print(" ", end="")

    # print hashes

            for z in range(0, i+1):

                print("#", end="")


    # print spaces

            for j in range(0, 2):

                print(" ", end="")

     # print hashes

            for z in range(0, i+1):

                print("#", end="")

        print();




if __name__ == "__main__":
    main()